utils/python/tests/scripts_regression_tests.py

```diff
@@ -1016,11 +1016,15 @@ def simple_test(self, manual_timing=False):
     ###########################################################################
     def test_save_timings(self):
     ###########################################################################
+        if CIME.utils.get_model() != "acme":
```

Comment:
You probably don't want to skip entire test, just skip the provenance_dir check at 1013.